core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -2053,13 +2053,19 @@ protected function getExtJSDirClasses()
      *
      * @return array
      */
-    protected function getListDirContextMenu()
+    protected function getListDirContextMenu($dirName)
     {
         $canSave = $this->checkPolicy('save');
         $canRemove = $this->checkPolicy('remove');
         $canCreate = $this->checkPolicy('create');
 
         $menu = [];
+        $menu[] = [
+            'text' => '<b>' . $dirName . '</b>',
```

Comment:
I consider that the appearance (including `font-weight`) should be controlled on the frontend and not the server.

Reply:
In an ideal world, yes, but in MODX the context menu is done via `<b></b>`.
It is worth redoing everywhere and from scratch, but this is another task for a separate PR.